### ISSUE TITLE
Class names

### ISSUE THREAD

**Issue by martoio** [Lufedi/prj-rev-bwfs-tea-cozy#4]
When working with class names comprised of multiple words, you should use hyphens to separate the words: ``tea-of-the-month``.

Another thing, this class name is too similar to the ``teas-of-the-month`` that you have a few lines down. Consider changing the ``teaofthemonth`` to something like: ``tea-of-the-month-heading``. Or something like that.

https://github.com/Lufedi/prj-rev-bwfs-tea-cozy/blob/33b2c4b73e5863c7e965d3b7def315fb6d7cb888/Tea%20Cozy/index.html#L31

**Comment by Lufedi** [comment]
You're right I had a mistake naming those classes, I'll try to use snake case now in all of them
